app/views/assignment/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>Creating New Assignment</h1>
+<h3><%= print_date(@week) %></h3>
+
+
+<%= form_for :assignment do |form| %>
```

Comment:
Form blocks might be the only place I ever use abbreviated variables. I'd use `f` here. I'm just sharing and changes are optional.